corehq/motech/repeaters/tests/test_dbaccessors.py

```diff
@@ -50,7 +67,6 @@ def setUpClass(cls):
             domain=cls.domain,
             succeeded=True,
             repeater_id=cls.repeater_id,
-            next_check=before,
```

Comment:
Why was this line removed/how was it breaking the tests?

Reply:
It is not valid to have a `next_check` date in SQL when `state == Success`.

https://github.com/dimagi/commcare-hq/blob/f8abe9215410b3af3561296a5aca8b2b6687d58f/corehq/motech/repeaters/models.py#L1366-L1373